api/src/main/resources/workbench-api.yaml

```diff
@@ -2339,6 +2339,21 @@ paths:
           schema:
             type: boolean
             default: false
+  "/v1/workspaces/{workspaceNamespace}/{workspaceId}/cohorts/{cohortId}/extractGenomesJob":
```

Comment:
Thinking further, I'm wondering whether this should be under dataset vs cohorts. Further down the road, users would be able to specify further criteria on the extraction, including potentially a subset of specific variant sites (or criteria). I think this will follow from the product design here, so this is fine for now - something to consider